exec-example.ts

```diff
@@ -0,0 +1,11 @@
+import k8s = require('@kubernetes/typescript-node');
```

Comment:
any idea what the `@kubernetes/` notation is? never seen that before

Reply:
Yeah, this is the organization notation from `npm.org`.